content/doc/administrate/log-management.md

```diff
@@ -148,7 +149,7 @@ And remove them if needed
 clever drain remove [--alias <alias>] <DRAIN-ID>
 ```
 
-If the status of your drain is shown as `DISABLED` without you disabling it, it may be because we  have not been able to send your logs to your drain endpoint or because the requests timed out after **25 seconds**.
+If the status of your drain is shown as `DISABLED` without you disabling it, it may be because we have not been able to send your logs to your drain endpoint or because the requests timed out after **25 seconds**.
```

Comment:
```suggestion
If the status of your drain appears as `DISABLED` without you disabling it, it may be because we have not been able to send your logs to your drain endpoint or because the requests timed out after **25 seconds**.
```